pycon/settings.py

```diff
@@ -25,7 +25,21 @@
 
 SITE_DATA_ROOT = DATA_DIR + '/site'
 
-DATABASES = {
+DATABASE_TYPE = os.environ.get('DATABASE')
+
+if DATABASE_TYPE == "postgres":
+    user = os.environ.get('DATABASE_USER')
+    password = os.environ.get('DATABASE_PASSWORD')
+    DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'ep2015_testing',
```

Comment:
I agree with you about one setting, but I have the experience where we can forget to change the settings during the deployment in production. but ok.